npm/tailcall/scripts/generate-packages.js

```diff
@@ -0,0 +1,66 @@
+import * as fs from "node:fs"
+import {dirname, resolve} from "node:path"
+import {fileURLToPath} from "node:url"
+
+const APP_VERSION = process.env.APP_VERSION
+
+const MANIFEST_PATH = resolve(fileURLToPath(import.meta.url), "../..", "package.json")
+
+const rootManifest = JSON.parse(fs.readFileSync(MANIFEST_PATH).toString("utf-8"))
+
+function generatePackages() {
+  const __filename = fileURLToPath(import.meta.url)
+  const __dirname = dirname(__filename)
+
+  let target
+
+  try {
+    target = resolve(__dirname, "../../../dist")
+  } catch {
+    console.log("[Publish] Error, no build is available")
+    process.exit(1)
+  }
+
+  const optionalPackages = {}
+
+  try {
+    console.log("[Publish] Read Built Files")
+    const files = fs.readdirSync(target)
+
+    for (const _file of files) {
+      const packageName = _file.replace("tailcall-", "")
+      const packageRoot = resolve(__dirname, "../..", packageName)
+
+      console.log(`[Publish] Copy ${_file} to ${packageRoot} `)
+      fs.copyFileSync(`${target}/${_file}`, `${packageRoot}/tailcall`)
+
+      const packageFileName = `${packageRoot}/package.json`
+
+      const packagefile = fs.readFileSync(packageFileName, {encoding: "utf-8"})
+
+      const packageJSON = JSON.parse(packagefile)
+
+      packageJSON["version"] = APP_VERSION || "latest"
```

Comment:
`APP_VERSION || "latest"` repeats several time and I think could be replaced by settings initially when defining the variable.

Or might be better to check that `env.APP_VERSION` is defined and not to publish with any defaults as it could be marker of some issues